core/data-plane-selector/data-plane-selector-core/build.gradle.kts

```diff
@@ -10,11 +10,7 @@
  *  Contributors:
  *       Microsoft Corporation - initial API and implementation
  *
- */
-
-val mockitoVersion: String by project
-
-plugins {
+ */plugins {
```

Comment:
there are some of these "missing newline" typos

Reply:
mmh, not only that, i also noticed that the latest rebase got mangled.